packages/react/src/components/ImageComponent/__stories__/ImageComponent.stories.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import './index.scss';
+import { object, text, withKnobs } from '@storybook/addon-knobs';
+import ImageComponent from '../ImageComponent';
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('ImageComponent', module)
+  .addDecorator(withKnobs)
+  .add('Default', () => {
+    const defaultImages = text(
+      'Default image:',
+      'https://picsum.photos/id/2/672/672'
+    );
+    const imageObject = object('Images Object:', [
+      'https://picsum.photos/id/2/320/160',
+      'https://picsum.photos/id/2/400/400',
+      'https://picsum.photos/id/2/672/672',
+    ]);
```

Comment:
image object should be an array of objects
```
image: [
  {
     minWidth: 0,
     url: .....
  },
 {
     minWidth: 320,
     url: .....
  },
 {
     minWidth: 1024,
     url: .....
  }
]
```

Reply:
updated the object with array of objects